### PR TITLE
Move account creation limits to the service account

### PR DESCRIPTION
**Pull request by ramtinms**
This PR adds a migration to move the account creation limit from FVM responsibility to service account responsibility.

### CODE REVIEW

File: cmd/util/ledger/migrations/account_creation_migration.go [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+	sth := state.NewStateHolder(st)
+	a := state.NewAccounts(sth)
+
+	address := "e467b9dd11fa00df"
+	err := migrateContractForAccount(a, address)
+	if err != nil {
+		return nil, err
+	}
+
+	key := "/storage/isAccountCreationRestricted"
```
Comment (ramtinms) [inline cmd/util/ledger/migrations/account_creation_migration.go:26]: Is this right key, consider we call the encode function later?

Comment (janezpodhostnik) [reply cmd/util/ledger/migrations/account_creation_migration.go:26]: It corresponds to this: https://github.com/onflow/flow-core-contracts/blob/ab9fccdc7204968fc943db418e145a5a9638faeb/contracts/FlowServiceAccount.cdc#L111

but not sure the string formatting is ok.

**Review by turbolent — APPROVED** [review]
Looks good!

**Comment by ramtinms** [comment]
Spork is over we don't need to merge this branch anymore

**Pull request closed without merging**